services/category.py

```diff
@@ -67,3 +68,13 @@ async def get_maximum_page():
             else:
                 return math.trunc(max_page / CategoryService.items_per_page)
 
+    @staticmethod
+    async def get_name(category_id) -> str:
+        async with async_session_maker() as session:
+            stmt = select(Category.name).where(Category.id == category_id)
+            category_name = await session.execute(stmt)
+            category_name = category_name.scalar()
+            if category_name is None:
+                return ""
```

Comment:
return None

Reply:
category = CategoryService.get_by_primary_key(subcategory_id)
category_name = category.name

Reply:
method returns str which is not None